Assignment2/GuillaumeChamp-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+  "@context :{
+    "my":"https://github.com/GuillaumeChamp",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "fullName": "rdfs:label",
+    "hometown": "rdfs:label",

```

Comment:
Not all properties should be transformed to rdfs:label.

Reply:
Can we not specify the type of a property or is it better to define them using a non-adapted one ?